tendrl/monitoring_integration/grafana/alert_dashboard.py

```diff
@@ -204,102 +202,118 @@ def create_resource_dashboard(
                 resource_json["dashboard"]["rows"] = []
                 resource_json["dashboard"]["rows"] = all_resource_rows
                 resource_json["dashboard"]["templating"] = {}
-                resp = alert_utils.post_dashboard(resource_json)
-                if resp.status_code == 200:
-                    msg = ("Alert dashboard for %s-%s is created " +
-                           "successfully") % (resource_name, new_title)
-                    logger.log("debug", NS.get("publisher_id", None),
-                               {'message': msg})
-                else:
-                    msg = "Alert dashboard upload failed for %s-%s" % \
-                          (resource_name, new_title)
-                    logger.log("debug", NS.get("publisher_id", None),
-                               {'message': msg})
+                return resource_json
```

Comment:
Why response status code check is removed? We will miss these debug messages as well of some failure.

Reply:
@shtripat i am not pushing dashbaord into grafana here, here dashbaord is created but not for all resource, only for the first resource I am populating json but for other resource row is duplicated, after that only i am pushing dashboard into grafana. here i am pushing and checking response https://github.com/Tendrl/monitoring-integration/pull/371/files#diff-08db395a1a58f2d111649d07a6495813R122

Reply:
ok